pkg/registry/styles/projects.less

```diff
@@ -1,8 +1,52 @@
 
 @tag-border: @link-color;
 
+table.project-body {
+    margin-top: 0px;
+
+    thead.th {
+        border-top:0px;
+        font-size: 12px;
+    }
+
+    tr {
+        display: table-row;
+        vertical-align: inherit;
+        border-color: inherit;
+    }
+
+    tr.listing-item {
+        border-top: 1px solid #eee;
+        border-bottom: 1px solid #eee;
+        cursor: pointer;
+    }
+}
+
+table.listing > thead td {
+    padding-top: 0px;
+}
+
+.project-panel {
+    padding-right: 25px;
+    padding-top: 0px;
+}
+
 table.projects-listing {
+    max-width: 2000px;
+    min-width: 85%;
+
+    tbody:hover {
+        background-color: #d4edfa;
+    }
+
+    tr.project-item {
```

Comment:
There is no mention of 'project-item' anywhehre outside this less file. I think this block is unnecessary.